docs/resources/interface_wireguard.md

```diff
@@ -0,0 +1,40 @@
+# mikrotik_interface_wireguard (Resource)
+Creates a Mikrotik interface_wireguard.
+
+!> This resource is supported for RouterOS v7+.
```

Comment:
It looks like these docs were edited by hand. They must be generated with the `tfplugindocs` tool. See my comment in the `mikrotik/resource_interface_wireguard.go` file on how to add the RouterOS v7 information to the autogenerated file.

Reply:
If the [default resource templating](https://github.com/ddelnano/terraform-provider-mikrotik/blob/f705437cb57bc15f760866df98e6c949ea98665b/templates/resources.md.tmpl) isn't appropriate, you can create resource specific templates as well (like the [bgp_peer](https://github.com/ddelnano/terraform-provider-mikrotik/blob/f705437cb57bc15f760866df98e6c949ea98665b/templates/resources/bgp_peer.md.tmpl) resource).